src/ApplicationInsights/ApplicationInsights.Autorest/test/utils.ps1

```diff
@@ -98,5 +118,5 @@ function setupEnv() {
 }
 function cleanupEnv() {
     # Clean resources you create for testing
-    Remove-AzResourceGroup -Name $env.resourceGroup
+    # Remove-AzResourceGroup -Name $env.resourceGroup
```

Comment:
Why was the remove command commented out?

Reply:
I have added the reason on line 71.

Reply:
Can we use a new resource group name? If some one needs to re-record the test case, still he cannot delete the resource group?